### PR TITLE
Add Capstone Branch

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
**In this PR, I did the following:**

- Created two active pages (home page and about page)
- Created the feature section and mobile menu dynamicly
- Added Loom vido link of capston project live demonstraion 
- Cleared All linter errors
- Updated the README file according the project changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      transform their
+      ideas into the reality that can contribute towards the development of the IT industry.
+    </p>
+
+    <p class="h1 date">2023.04.14(MON) ~ 2023.06.15(FRI)</p>
+    <p class="location ">@National Conference Hall of Kabul University, and more</p>
+  </section>
+
+  <section class="main  mt-5 px-3 py-5">
+    <h3 class="main-program text-center text-white fw-bold">Main Program</h3>
```
Comment (cynthiainga) [inline index.html:76]: - [ ] As you can see in the design this section should contain five articles but you have only 4 articles, kindly make sure you have 5 articles 😉 😉 

Your design

Screenshot 2023-03-23 at 18 10 55

Original design

Screenshot 2023-03-23 at 18 12 31

File: index.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    ham.classList.remove('change');
+    ul.classList.remove('change');
+  } else {
+    ham.classList.add('change');
+    ul.classList.add('change');
+  }
+});
+
+// Speakers data
+const speakers = [
```
Comment (cynthiainga) [inline index.js:16]: - [ ] This section should contain 6 individual articles on the desktop version but only 4 are shown. Please kindly fix that 🙃 🙃 

Your design

Screenshot 2023-03-23 at 18 26 37

Original design

Screenshot 2023-03-23 at 18 27 47

File: about.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+    <p class="topic-description contact-us text-center px-5 mt-5 pt-5">
+      Please contact us via following Email for any
+      further questions about Web Development company
+    </p>
+    <p class="text-center mail"><a href="mailto:Web-Development@outlook.com">Web-Development@outlook.com</a></p>
+  </section>
+
+  <section class="card p-lg-3 pb-lg-5 border-1 m-0">
+    <h2 class="logo-text text-center px-4 py-4 h4">Web Development Company Bootcamp <br>2023 Logo</h2>
```
Comment (cynthiainga) [inline about.html]: - [ ] After the title we have a divider, please kindly add it 🙃 🙃 

Your design

Screenshot 2023-03-23 at 18 55 06

Original design

Screenshot 2023-03-23 at 18 55 53

File: about.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    <div class="d-lg-flex my-5 px-3 justify-content-center justify-content-lg-center">
+      <div class="gradPhoto px-1 mt-4 me-lg-2 text-center align-items-center">
+      </div>
+      <div class="gradPhotoes px-1 mt-4 ms-lg-2 text-center align-items-center">
+      </div>
+    </div>
+    </div>
+    </div>
+
+  </section>
```
Comment (cynthiainga) [inline about.html]: - [ ] This section should contain a header, a divider, and a line of text that you miss kindly add them 🙏 🙏 

Your design

Screenshot 2023-03-23 at 18 58 19

Original design

Screenshot 2023-03-23 at 19 17 02

File: README.md [hunk 5]
```diff
@@ -1 +1,154 @@
...
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Mobile first Home page ]**
+- **[About page]**
+
+
+- [🚀 Live Demo](#live-demo)
```
Comment (cynthiainga) [inline README.md]: - [ ] As required in the project you should deploy your project using the Github page and then add the live demo to your README file. Also after the deployment kindly makes sure your website works properly, and that all pages, links, and images work as expected 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Jawad-Nazari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct GitHub flow ✔️ 


## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- I noticed that some of your commit messages are not capitalized and not in the imperative tense. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8), please kindly remember this for your next commit messages 😉 😁 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
Hi @Jawad-Nazari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct GitHub flow ✔️ 
- [ ] Good design ✔️ 
- [ ] Descriptive PR ✔️ 

## Optional suggestions

- [OPTIONAL] As mentioned by the previous reviewer the commit message should be capitalized and in the imperative tense, please kindly remember that for your next commit 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**